### PR TITLE
Features/custom loss torch

### PR DESCRIPTION
**Pull request by AstrakhantsevaAA**

### CODE REVIEW

File: src/data_utils/dataset.py [hunk 1]
```diff
@@ -99,14 +65,25 @@ def __getitem__(self, index):
...
-        label = int(label) - 1
-        label = torch.tensor(label, dtype=torch.long)
+
+        label_scaled = 0.0
+        label = 0.0
+
+        if not self.inference:
+            label = self.data[net_config.label_column].iloc[index]
+            if net_config.label_column == "severity":
+                label_scaled = int(label) - 1
```
Comment (b0nce) [inline src/data_utils/dataset.py:83]: Why -1?

File: src/train/classificator/sampler.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    df_full = dataset.df_full
+    df_train = df_full[df_full["split"] == "train"]
+    df_val = df_full[df_full["split"] == "validation"]
+    df_test = df_full[df_full["split"] == "test"]
+
+    weights_train = calculate_weights(df_train)
+    weights_val = calculate_weights(df_val)
+    weights_test = calculate_weights(df_test)
+
+    weights_adv = {k: v / weights_val[k] for k, v in weights_train.items()}
```
Comment (b0nce) [inline src/train/classificator/sampler.py:26]: But then you should re-train the whole model to predict on test
Seems strange, we should discuss this

Comment (b0nce) [reply src/train/classificator/sampler.py:26]: resolved

**Review by b0nce — APPROVED** [review]

File: src/data_utils/dataset.py (hunk 1 above)
Comment (AstrakhantsevaAA) [reply src/data_utils/dataset.py:83]: lables are started from 1. Just wanted transform it in [0,4] diapason
